### DISCUSSION TITLE
[@types/hotwired__turbo] Property 'drive' does not exist on type 'TurboSession'.

### DISCUSSION THREAD

**Discussion by DawidJanczak** [DefinitelyTyped/DefinitelyTyped#69691] [Issues with a @types package]
Hello,

It seems this is missing, we should be able to do `Turbo.session.drive = false` as per docs https://turbo.hotwired.dev/handbook/drive

[a comment by an automated account is not shown]

**Comment by G-Rath** [comment]
This has been addressed via #74287
